src/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  font-family: Roboto, Arial, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-image {
+  padding: 16px 0;
+  margin: 0 auto;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+  object-fit: fill;
+}
+
+.product-description {
+  padding: 18px 0 4px;
+}
+
+.product-description__link {
+  font-size: 12px;
+  line-height: 12px;
+  font-weight: 500;
+  color: #060b35;
+  text-decoration: none;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 0;
+}
+
+.product-rate__stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-size: 13px 13px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star--active {
```

Comment:
Replace this logic. You need to add a class with the numbers to the wrapper `div`, and in CSS make logic of stars using this number.